metron-platform/metron-writer/src/test/java/org/apache/metron/writer/BulkWriterComponentTest.java

```diff
@@ -161,9 +165,12 @@ public void writeShouldThrowExceptionWhenHandleErrorIsFalse() throws Exception {
   @Test
   public void writeShouldProperlyHandleWriterException() throws Exception {
     Throwable e = new Exception("test exception");
-    MetronError error = new MetronError()
+    MetronError expectedError1 = new MetronError()
             .withSensorType(Collections.singleton(sensorType))
-            .withErrorType(Constants.ErrorType.INDEXING_ERROR).withThrowable(e).withRawMessages(Arrays.asList(message1, message2));
+            .withErrorType(Constants.ErrorType.INDEXING_ERROR).withThrowable(e).withRawMessages(Collections.singletonList(message1));
```

Comment:
Small nit.  While we are in here, do you think it is easier to read with newlines?  
```suggestion
            .withErrorType(Constants.ErrorType.INDEXING_ERROR)
            .withThrowable(e)
            .withRawMessages(Collections.singletonList(message1));
```

Reply:
I agree.  Done.